src/schnetpack/representation/painn.py

```diff
@@ -211,6 +237,11 @@ def forward(self, inputs: Dict[str, torch.Tensor]):
         idx_j = inputs[properties.idx_j]
         n_atoms = atomic_numbers.shape[0]
 
+        #inputs needed for charge and spin embedding
```

Comment:
total charge and spin multiplicity is not always given right? This would through an error